docs/components/feedback_banner.md

```diff
@@ -0,0 +1,31 @@
+
+# Feedback banner
+
+To add the **default** feedback banner just paste this in your `home_page.html` template.
+```django
+{% block feedback_banner %}
+{% include 'wagtailnhsukfrontend/feedback_banner.html' %}
+{% endblock %}
+```
+
+| Option | Description | Default |
+| ------ | ----------- | ------- |
+| `title` | Title to display for the for the banner| `Help us make the NHS website better` |
```

Comment:
I don't think we should give any default values here.
These defaults are only really useful for the NHS website. We could come up with some more generic defaults maybe?